### ISSUE TITLE
Shop overlay does not support tourism=hotel and friends

### ISSUE THREAD

**Issue by mss** [streetcomplete/StreetComplete#5152]
**Use case**
Hotels, Bed & Breakfasts, etc. can't be added (and verified) via the app. Admittedly they aren't really shops but neither are some other amenities like kindergartens. Small guest houses are often located where small shops could be and it would be useful if one could just add them like the rest without switching apps.

**Proposed Solution**
I think these types of amenities should be available:

* tourism=hotel
* tourism=motel
* tourism=hostel
* tourism=guest_house
* maybe others

**Comment by westnordost** [comment]
Related https://github.com/streetcomplete/StreetComplete/issues/5065#issuecomment-1589524501 and following comments.

So, in that regard, since hotels, motels, ... generally do not occupy shop spaces but rather either are purpose-built (many hotels) or occupy otherwise residential/office space, they are not included in the list. It should not be surprising to not find these POIs in the shop overlay, given its name. Retirement homes, hospitals, schools etc. are also not included.

**Comment by mss** [comment]
Fair enough, I can continue using EveryDoor for these POIs.

But I'd like to point out that nowadays Churches are also included in the Shop Overlay. As are Kindergartens/Preschools and Corporate Offices. None of these are Shops and follow similar occupation patterns in dense metropolitan areas as Guest Houses, BnBs, and small hotels.

An example where this would have been useful is the changeset https://www.openstreetmap.org/changeset/138065994 in which I switched to EveryDoor to verify a Guest House and removed another. In the latter case the space was repurposed as a shop again. I added the seemingly missing shop (and created a duplicate POI) and only later discovered that it already existed but just wasn't shown in SC when I checked that street.

**Comment by westnordost** [comment]
Hm, well as written, a line has to be drawn which POIs to include and which not _somewhere_, right?

**Comment by mnalis** [comment]
> line has to be drawn which POIs to include and which not somewhere, right?

Probably, but it does needs some work to draw that line, as OSM tagging is a kind of a mess. Probably it would be easiest to copy it from project that has already put in the work to do it? 

For example, Every Door has `amenities` mode (similar to `Shop overlay` in SC) and `micromapping` mode (similar to proposed [street furniture](https://github.com/streetcomplete/StreetComplete/issues/4912) overlay in SC). The line drawn there seems pretty good to me.

AFAICT, basically it has a [whitelist/blacklist](https://github.com/Zverik/every_door/blob/main/lib/helpers/good_tags.dart#L83) list of `real amenities` (i.e. places where people would want to get in to buy something or get some service or to sleep etc.)  and everything else (like streetlamps, benches, etc) falls under `micromapping`.

See all `const` in https://github.com/Zverik/every_door/blob/main/lib/helpers/good_tags.dart

e.g. `kMainKeys` defines (among others) that `amenity=*` is **"real amenity"**, _**unless**_ its value is present in `wrongAmenities` (like `amenity=bench`, `waste_basket`, `telephone`...) - that repeats for other keys too like `tourism=*`, `leisure=*` etc.

---

Maybe @Zverik would be interested to have that inclusion/exclusion lists for _"amenities vs. micromapping"_ transformed into some generic language-independent format like JSON, so it can then easily be shared by multiple independent projects? Or perhaps a flag might be introduced into `id-tagging-presets` to indicate if the tag is "real" amenity?

**Comment by westnordost** [comment]
I am just going to tag @tyrasd here too because IMO such a distinction of shops/amenities/places vs. micro-mapped things would be useful for any consumer of id tagging presets: All editors, but also general-purpose renderers, I'd say. So it sounds useful to have this information directly in the presets themselves, no?

As for naming, I'd suggest **"place"** vs **"furniture"**:
Hospital, hairdresser, shoemaker, office, church, hotel, café, ... are places.
Street lamps, benches, ATMs, vending machines, bicycle parkings, defibrillators, bollards, traffic signs, trees, ... are not places.

**Comment by mnalis** [comment]
> As for naming, I'd suggest "place" vs "furniture":
Hospital, hairdresser, shoemaker, office, church, hotel, café, ... are places.
Street lamps, benches, ATMs, vending machines, bicycle parkings, defibrillators, bollards, traffic signs, trees, ... are not places.

makes sense to me.
> As far as definition goes, I guess what (most?) places have in common is that you can go inside.

Most, but not all. For example, I'd consider `amenity=fast_food` (or `shop=kiosk`) as a _"place"_, even if  I cannot go inside, but they would just take my order and give it to me to take away.

**Comment by 1ec5** [comment]
When gathering statistics locally, I’ve used the rule of thumb that a POI is something that a user might want to search for, thus anything that typically has a name. From that I would exclude anonymous address points, roads, and boundaries, but otherwise this heuristic has worked well for me. I include bus stops and ATMs (named after the operator) based on local custom, which might differ in other places.

**Comment by westnordost** [comment]
True, good observation. But of course, `place=*` also has a name, so (can) have buildings, artworks, roads, landuses etc.
Then on the other hand, small shops (like kiosks) often do not have a (signed) name.

Also, as far as I know, it is not possible to search for iD presets that **could have** a name, because the `name` field is universal, i.e. addable to all presets.

**Comment by Zverik** [comment]
To me (and Every Door mostly), POI is a venue manned by people or providing paid services. So, shops, churches, vending machines and parcel delivery machines, but not post boxes or garbage bins.

**Comment by 1ec5** [comment]
If we don’t count post boxes and garbage bins as POIs, how could we ever hope to match Overture’s advertised POI count? 😛

The reality is probably that the true definition of a POI depends on the audience: what a consumer considers a POI may differ from what a government considers a POI, and a data vendor would have their own definition too. So I think there’s some room for StreetComplete to have its own opinion.

**Comment by Zverik** [comment]
Well all of these are POI. POI are tagged points. Traffic lights, trees, stop signs can be considered POI.

What I meant are things to pay special attention to, which most likely have opening hours, payment options, names obviously, can be located inside buildings etc etc.

**Comment by 1ec5** [comment]
> POI are tagged points.

I don’t think anyone here is referring to OSM’s technical definition of a POI. We’re all grasping at a real-world definition of a POI but finding that there are multiple definitions. This may be why some non-user-facing systems use alternative terms like “facility” and “venue” that they can define for their own purposes. For a user-facing purpose, “POI” is still somewhat more intuitive. I happen to like the elegance of “whatever someone would want to see in search results, other than a plain address”.

**Comment by mnalis** [comment]
> I don’t think anyone here is referring to OSM’s technical definition of a POI
> For a user-facing purpose, “POI” is still somewhat more intuitive. 

True, but the clash of meaning is still unfortunate. So definitely worth to explicitly clarify it in this discussion.

And using non-ambiguous word might be better idea even for end-users (as you mention in “facility” and “venue” examples). 

> I happen to like the elegance of “whatever someone would want to see in search results, other than a plain address”.

Or _"something user might want to enable audio announcement for in navigation app"_. 

e.g. you may want your navigation app to announce _"You'll be passing fast food McDonalds in 100m"_ or _"You'll be passing Hotel California in 100m"_ or _"You'll be passing petrol station in 100m"_ (depending on whether you're needing food or rest or fuel, of course) but you ***don't*** really want it to announce _"You'll be passing street lamp"_ or _"You'll be passing waste basket"_ (or even _"You're passing housenumber 42"_ )

**Comment by maurerle** [comment]
I came to this issue because I wanted to map a Hostel/Hotel too - would be very cool to have this option.

I like @mnalis approach here https://github.com/streetcomplete/StreetComplete/issues/5152#issuecomment-1671096151
But it also would be good to just add the four initially mentioned keys to the current list

**Comment by westnordost** [comment]
The new comment/description for what is a "shop" (now "place") is the following:

> ### **Map features like shops or amenities that usually have a name and can be entered.**
>
>  To demarcate this category from others, the following are not included:
> - things that are rather structures or buildings than places (e.g. communication towers, train station)
> - landuses, outside-things (golf courses, zoos, marketplaces, car parking)
>
>  **Note**: When this function is modified, please follow update instructions in:  https://github.com/mnalis/StreetComplete-taginfo-categorize/blob/master/README.md
>
>  **Important**: The contribution policy here is that only things may be added if an iD preset for it exists. This policy exists in order to reduce effort to maintain this list, i.e. we don't want to check, weigh and balance requests in parallel to iD maintainers (in terms of notability, it being unambiguous, consensus etc.)

So, while the list of things included in this overlay has been extended by things that are purpose-built - schools, universities, stadiums, hotels, guest houses, hostels, motels, police stations, fire brigade stations, bowling alleys, hospitals, cinemas, conference centers, townhalls, etc., what is still omitted are things that are rather like landuses, such as parks, gardens, golf courses, but also marketplaces, miniature golf courses, zoos, theme parks, military bases etc. 

So, some things that may be called "places" are still missing from this overlay even though they can be entered, but I found it weird to add it into this category, or what do you think? Here's a list of things I skipped:

- **leisure=** beach_resort, disc_golf_course, garden, golf_course, horse_riding, marina, miniature_golf, nature_reserve, park, pitch, playground, resort, schoolyard, shooting_ground, sports_centre, summer_camp, swimming_area, track, water_park
- **tourism=** camp_pitch, camp_site, caravan_site, picnic_site, theme_park, zoo
- **amenity=** animal_breeding, boat_storage, building_yard, bus_station, driver_training, ferry_terminal, festival_grounds, grave_yard , marketplace, parking, traffic_park, recycling,   recycling_type=centre, refugee_site , waste_transfer_station, waste_dump_site
- **military=** airfield, base, academy, ...

See changes here: https://github.com/streetcomplete/StreetComplete/pull/5373/commits/6eb6ddf9c278e66d913bb8ed1beb48017fdb7cc0

**Comment by westnordost** [comment]
Also ping @mnalis because I updated the list of things that should be classifed as "shop" (now "place").
